Item/Item.js

```diff
@@ -85,30 +141,159 @@ const ItemContent = kind({
+		 * Called with a reference to the root component.
+		 *
+		 * @type {Object|Function}
+		 * @public
+		 */
+		componentRef: EnactPropTypes.ref,
+
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * `item` - The root class name
+		 * `slotBefore` - The slot (container) preceding the text of this component
+		 * `slotAfter` - The slot (container) following the text of this component
+		 * `selected` - Applied to a `selected` button
+		 *
+		 * @type {Object}
+		 * @public
+		 */
 		css: PropTypes.object,
+
+		/**
+		 * Applies a disabled style and the control becomes non-interactive.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		disabled: PropTypes.bool,
+
+		/**
+		 * Applies inline styling to the item.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		inline: PropTypes.bool,
+
+		/**
+		 * The label to be displayed along with the text.
+		 *
+		 * @type {Node}
+		 * @public
+		 */
 		label: PropTypes.node,
+
+		/**
+		 * The position of the label relative to the primary content, `children`.
+		 *
+		 * @type {('above'|'after'|'before'|'below')}
+		 * @public
+		 */
 		labelPosition: PropTypes.oneOf(['above', 'after', 'before', 'below']),
+
+		/**
+		 * Determines what triggers the marquee to start its animation.
+		 *
+		 * @type {('focus'|'hover'|'render')}
+		 * @public
+		 */
+		marqueeOn: PropTypes.oneOf(['focus', 'hover', 'render']),
+
+		/**
+		 * Applies a selected style to the component.
+		 *
+		 * @type {Boolean}
+		 * @private
+		 */
 		selected: PropTypes.bool,
+
+		/**
+		 * The size of the item.
+		 *
+		 * @type {('large'|'small')}
+		 * @default 'large'
+		 * @private
+		 */
+		size: PropTypes.oneOf(['small', 'large']),
+
+		/**
+		 * Nodes to be inserted after `children`.
+		 *
+		 * For LTR locales, the nodes are inserted to the right of the primary content. For RTL
+		 * locales, the nodes are inserted to the left. If nothing is specified, nothing, not even
+		 * an empty container, is rendered in this place.
+		 *
+		 * @type {Node}
+		 * @public
+		 */
 		slotAfter: PropTypes.node,
+
+		/**
+		 * Nodes to be inserted before `children` and `label`.
+		 *
+		 * For LTR locales, the nodes are inserted to the left of the primary content. For RTL
+		 * locales, the nodes are inserted to the right. If nothing is specified, nothing, not even
+		 * an empty container, is rendered in this place.
+		 *
+		 * @type {Node}
+		 * @public
+		 */
 		slotBefore: PropTypes.node
 	},
+
+	defaultProps: {
+		labelPosition: 'below',
+		size: 'large'
+	},
+
 	styles: {
 		css: componentCss,
 		publicClassNames: ['item', 'label', 'slotAfter', 'slotBefore']
 	},
+
 	computed: {
-		className: ({selected, styler}) => styler.append({selected})
+		className: ({selected, centered, size, styler}) => styler.append(
+			{
+				selected,
+				centered
+			},
+			size
+		),
```

Comment:
I think some of sandstone/Item structure didn't sync in this PR.
`hasLabel` should be placed here, not in `ItemContent`.
Item.modules.less should have the proper structure along with this change.
Would you take a look, please?

Reply:
You're right, I missed this one. `hasLabel` is placed in `ItemBase` now. Thanks for noticing!